src/contour/Actions.h

```diff
@@ -151,6 +152,23 @@ using Action = std::variant<CancelSelection,
                             SwitchToTabLeft,
                             SwitchToTabRight>;
 
+template <typename T>
+concept RepeatableActionConcept = crispy::one_of<T,
```

Comment:
Not sure what is better, either note the repeatable ones, or the ones that should be protected from key-repeat. :thinking: